Formula/spades.rb

```diff
@@ -14,14 +14,16 @@ class Spades < Formula
 
   depends_on "cmake" => :build
   depends_on "libomp"
+  depends_on "python@3.8"
 
   uses_from_macos "bzip2"
   uses_from_macos "ncurses"
-  uses_from_macos "python@2"
   uses_from_macos "readline"
   uses_from_macos "zlib"
 
   def install
+    inreplace "spades.py", "/usr/bin/env python", Formula["python@3.8"].opt_bin/"python3"
```

Comment:
Can we use shiny new [`rewrite_python_shebang`](https://github.com/homebrew/brew/blob/30c4383e86e1f9c8390116171f95cd464ad903c4/Library/Homebrew/language/python.rb#L90-L99) here?

Reply:
Absolutely! Done.